utility/src/main/AndroidManifest.xml

```diff
@@ -1,3 +1,5 @@
-<manifest package="org.oppia.util">
-
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+  package="org.oppia.util">
+  <uses-sdk android:minSdkVersion="19"
+    android:maxSdkVersion="28" />
```

Comment:
We should specify a target, but not a max. Otherwise when we upgrade the version the app targets we’ll have compatibility issues. 

Ditto for the other manifest.

Reply:
Sounds good!